hbase-client/src/main/java/org/apache/hadoop/hbase/RegionLocations.java

```diff
@@ -38,6 +38,9 @@
 
   private final int numNonNullElements;
 
+  // For Meta Replica LoadBalance mode, this indicates which meta replica the locations come from.
+  private int fromMetaReplicaId;
```

Comment:
Do we have other ways to pass this infromation? I do not think this is a good way to go as we create RegionLocations everywhere in code, not only at client side when locating regions. If there is no other simple way, I suggest we just remove the fromReplicaId parameter in the selector interface, as now the simple selector does not need it. The interface is IA.Private so we can change it at any time.

Reply:
Ok, going to remove the replica id for now. We can go back to it later when it is needed.